### PR TITLE
Add paper parameters

### PR DESCRIPTION
**Pull request by neithernut**
This PR adds a section about parameters for the data structure.
It's not yet quite complete (concrete thresholds for resizing missing), but it should suffice for now.

### CODE REVIEW

**Comment by neithernut** [comment]
Solves issues #179 and #180.

File: paper/parameters.tex [hunk 1]
```diff
@@ -1,2 +1,207 @@
...
+        height would be an acceptable upper limit for the height of an AVL.
+        This suggests that, for the overall construct to yield optimal
+        performance, the greater part of the elements should reside in AVLs of
+        a height less or equal than $4$, if $m=64$. For optimal performance, we
+        want the height of each AVL be close but not (much) greater than
+        $h_{opt}=4$.
+
+        Consider an instance of the data structure described in this document.
+        The buckets of the hash table can be grouped into three categories by
+        the height of the AVLs fo which they consist:
```
Comment (matthiasbeyer) [inline paper/parameters.tex]: Typo here. "fo which"

**Comment by matthiasbeyer** [comment]
Awesome. I do not understand most of it, but I think there are no typos (other than the mentioned one) or syntax errors in there.

I would like you to split up the `.mp` file to only generate `.1` files, so we don't have to bother with the `.2` extension!

**Comment by neithernut** [comment]
They share common code.

**Comment by matthiasbeyer** [comment]
Well, okay. Then please fix up the typo and do your rebasing if neccessary.

**Comment by neithernut** [comment]
Done.

**Pull request merged**